tests/lib/pkgdb.sh

```diff
@@ -191,6 +191,9 @@ distro_install_local_package() {
 }
 
 distro_install_package() {
+    orig_xtrace=$(set -o | grep xtrace | cut -f 2)
```

Comment:
Further reading `bash` manpage, you could do do `[[ -o xtrace ]] && orig_xtrace=on`

Reply:
Yeah but this works in shell and bash, safer to copy-paste around :)

Reply:
Ah, then `set -o | awk '/xtrace / { print $2 }'` is one fork less.